Datastore/src/Query/Aggregation.php

```diff
@@ -36,6 +36,16 @@ class Aggregation
      */
     private const TYPE_COUNT = 'count';
 
+    /**
+     * Default placeholder for all sum aggregation props.
+     */
+    private const TYPE_SUM = 'sum';
```

Comment:
In class comment, add that 'Aggregation considers non existing property name as an empty result set'

Reply:
Makes sense. Added